fem/gslib.cpp

```diff
@@ -184,9 +184,9 @@ void FindPointsGSLIB::FindPoints(const Vector &point_pos,
    gsl_ref.SetSize(points_cnt * dim);
    gsl_dist.SetSize(points_cnt);
 
-   auto xvFill = [&](const double *xv_base[], unsigned xv_stride[], int dim)
+   auto xvFill = [&](const double *xv_base[], unsigned xv_stride[], int dim_)
```

Comment:
Can we directly use the member variable `dim` in this function and get rid of `dim_` completely?

Reply:
done in [f60272a](https://github.com/mfem/mfem/pull/3750/commits/f60272ab0bb29583de9e4c1479f5bb1a8e2dbd42)